releases/data-center/syskit-point-2024-1/point-2024-1-41-release-note.md

```diff
@@ -0,0 +1,95 @@
+---
+description: >-
+  This article lists improvements and bug fixes in Syskit Point version 2024.1.41
+---
+
+# Syskit Point 2024.1.41
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2024.1.41
+
+**Build number:** 74
```

Comment:
ipak smo imali noviji build - ono što je Matija H. pushao .. tako da treba biti 2024.1.41.**75** @katperich @imesaric